Section 3.md

```diff
@@ -107,9 +107,9 @@ While usually the larger corporations devise their own power grid and manage the
 
 - **Morpheus Cyberkinetics**
 
-Originally a small company focused on research of the interface of the human mind to cybernetic parts, founded just prior to the explosion in robotics technology. This company produced the first of the truly high-end cybernetics, interfacing directly with the human nervous system, even giving a very rudimentary sense of touch. There are some rumors that Zeng-Hu had seized their results through corporate espionage, quickly producing their own line of prosthetics that eclipsed Morpheus� own.
+Founded just prior to the explosion in robotics technology, Morpheus was originally a smaller company focused on research of the interface designed for human mind to cybernetic connection. Eventually, Morpheus came to produce the first high-end cybernetic tech, interfacting directly with human nervous system, even giving a rudimentary sense of touch. There are rumors that Zeng-Hu had seized their results through corporate espionage, quickly producing their own line, but is unclear if those rumors were true.
 
-Morpheus, however, had one breakthrough that is entirely attributed to it, the �MMI�. This device not only allowed for people to be �revived� by having their brains transplanted into it, it also allowed them to access and interface with many custom-built robots and other devices.(However, at the time, it was most common to find cyborgs in modified drone chassis, as a proper chassis was simply too expensive for many.)
+Later in years, Morpheus came to yet another break through, this time fully attributed to them, with the MMI. The MMI not only allowed for people to be revived through brain transplants, but also allowed for them to access and interface with custom-built machinery, such as cyborg equipment. However, at the time, it was far more common to find MMIs in modified drone chassis, as the proper cyborg chassis were too expensive for most.
```

Comment:
It's not incredibly recent- still mid 24th century, IIRC.

And it's less a sake of size, more a sake of quality and how fit it is for humans, I believe.